build/build.yml

```diff
@@ -178,20 +178,6 @@ steps:
       **\Microsoft.Artifacts.Authentication.pdb
     TargetFolder: '$(Build.ArtifactStagingDirectory)\symbols'
 
-- ${{ if eq(parameters.sign, 'true') }}:
-  - task: PublishSymbols@2
-    displayName: Publish symbols
-    inputs:
-      SymbolsFolder: '$(Build.ArtifactStagingDirectory)\symbols'
-      SearchPattern:  '**\*.pdb'
-      IndexSources: false
-      PublishSymbols: true
-      SymbolServerType: TeamServices
-      SymbolsProduct: 'artifacts-credprovider'
-    env:
-      ARTIFACTSERVICES_SYMBOL_ACCOUNTNAME: 'microsoft'
-      ARTIFACTSERVICES_SYMBOL_USEAAD: 'false'
-
 - ${{ if eq(parameters.publish, 'true') }}:
   - task: PublishPipelineArtifact@1
     displayName: 'Upload symbols'
```

Comment:
If the pipeline artifact has symbol files included in it would say also remove this one then?

Reply:
Then I think all the parameters.publish usage is gone and we can remove that from all the other yamls

Reply:
good point removed symbols upload and removed unused publish parameter